### ISSUE TITLE
Size of parameters in Example 6.1

### ISSUE THREAD

**Issue by dmitrijsk** [uu-sml/sml-book-page#33]
**Location**
Please provide:
1. Version (bottom left page): November 22
2. Page number: 98
3. Line/equation number End of example 6.1.

**Describe the mistake**
W(2) has dimensions 100x10. 
b(2) missing in the calculation of the total number of parameters.

**Suggested change**
There is an input layer with 784 variables. W(1) maps these 784 into 200 hidden units in the hidden layer. The next layer is an output layer with 10 logits. Therefore, W(2) should map 200 hidden units into 10 output logits. I think the dimensions of W(2) should be 200x10 (not 100x10). Also, it seems that in the calculation of the total number of parameters the b(2) offset vector with 10 parameters is missing. However, the final number 159,010 is correct.

**Comment by nikwa** [comment]
Well spotted! You are all correct. Will be updated in next version.
